css-overflow-4/Overview.bs

```diff
@@ -1123,12 +1146,12 @@ Fragmentation of Overflow: the 'continue' property</h3>
 
 	Additionaliy,
 	for compatibility (see [[#webkit-line-clamp]]),
-	when the [=computed value=] of the 'continue' property is ''discard''
+	when the [=computed value=] of the 'continue' property is ''collapse'' or ''-webkit-collapse''
```

Comment:
```suggestion
	when the [=computed value=] of the 'continue' property is '''collapse'', ''-webkit-collapse'', or ''discard''
```

Strictly speaking, I think it is not necessary to list the -webkit- value here, since it is defined "to behaves identically to collapse". But I agree it is clearer to include it anyway.

I suggest including ''discard'' as well. If implementations ever support both, that makes it easier to switch between the two.